##CLOUD-SQL-CONFIGURATIONS
  spring.cloud.gcp.sql.database-name=vcpdb
  spring.cloud.gcp.sql.instance-connection-name=learning-project1-352200:us-central1:my-innstance-poig
  # So app starts despite "table already exists" errors.
  spring.datasource.continue-on-error=true
  # Enforces database initialization
  spring.datasource.initialization-mode=always
  spring.datasource.username=root
  spring.datasource.password=Onanan1956Jc_24.01.2020
  spring.cloud.gcp.project-id=learning-project1-352200
  spring.cloud.gcp.credentials.location=file:learning-project1-352200-932c250f9c1f.json
  
  spring.jpa.properties.hibernate.dialect= org.hibernate.dialect.MySQL5InnoDBDialect
  spring.jpa.hibernate.ddl-auto= update

